docs/onebusaway-gtfs-transformer-cli.md

```diff
@@ -270,6 +271,26 @@ You can remove a specific entity from a feed.
 
 Note that removing an entity has a cascading effect.  If you remove a trip, all the stop times that depend on that
 trip will also be removed.  If you remove a route, all the trips and stop times for that route will be removed.
+
+#### Retain Up From Polygon
```

Comment:
I must admit that I find the terminology of "retaining up" and "retaining down" confusing. What do you mean by that?

Reply:
We did'nt found a better term. This one is direct from the existing retain algotithm that seems very stable.
"Up" means that the algorithm will traverse up the entity dependency tree by applying retainUp to each of them. Starting from the stop, retainUp will be applied to the stop_times referencing this stop, then to the trips, and so on.
Then, when the base of the tree is reached, it applies retainDown to all the traversed entities. Therefore, all the trips of the route and then all the stop_times of each trip will be tagged as "retain".
Entities not marked as "retain" will be deleted.
The goal of the strategy is to retain all entities that are directly or indirectly linked to the area.

We will add this brief explanation to the documentation.

Reply:
okay, that's fine.